diff --git a/ElementsKernel/CMakeLists.txt b/ElementsKernel/CMakeLists.txt
index b2a8d1a..1203817 100644
--- a/ElementsKernel/CMakeLists.txt
+++ b/ElementsKernel/CMakeLists.txt
@@ -116,13 +116,6 @@ elements_add_unit_test(BackTrace tests/src/BackTrace_test.cpp
                        EXECUTABLE BackTrace_test
                        LINK_LIBRARIES ElementsKernel TYPE Boost)
 
-#-----------------------
-# System_test
-elements_add_unit_test(System tests/src/System_test.cpp
-                       EXECUTABLE System_test
-                       LINK_LIBRARIES ElementsKernel TYPE Boost)
-
-
 #-----------------------
 # GetEnv_test
 elements_add_unit_test(GetEnv tests/src/GetEnv_test.cpp
@@ -212,4 +205,4 @@ elements_add_test(PatchVersionProject
 elements_add_test(CMakeOverride 
                   COMMAND cmakeOverride_test  
                   ENVIRONMENT CMAKE_PREFIX_PATH+=${CMAKE_SOURCE_DIR}/cmake
-                  LABELS Creation Override)
\ No newline at end of file
+                  LABELS Creation Override)
